main.sh

```diff
@@ -3,22 +3,22 @@
 #Run daos tests like IOR/MDtest and self_test(cart)
 #----------------------------------------------------
 
-#SBATCH -J test_daos1           # Job name
+#SBATCH -J daos1                # Job name
 #SBATCH -o stdout.o%j           # Name of stdout output file
 #SBATCH -e stderr.e%j           # Name of stderr error file
 #SBATCH -A STAR-Intel           # Project Name
-#SBATCH -p skx-dev              # Queue (partition) name
-#SBATCH -N 4                    # Total # of nodes
-#SBATCH -n 192                  # Total # of mpi tasks (48 x  Total # of nodes)
-#SBATCH -t 1:00:00             # Run time (hh:mm:ss)
-#SBATCH --mail-user=first.last@intel.com
+#SBATCH -p development          # Queue (partition) name
+#SBATCH -N 5                    # Total # of nodes
+#SBATCH -n 280                  # Total # of mpi tasks (56 x  Total # of nodes)
+#SBATCH -t 00:30:00             # Run time (hh:mm:ss)
+#SBATCH --mail-user=<first.last>@intel.com
 #SBATCH --mail-type=all         # Send email at begin and end of job
 
 #Parameter to be updated for each sbatch
-DAOS_SERVERS=2
+DAOS_SERVERS=4
 DAOS_CLIENTS=1
 ACCESS_PORT=10001
-DAOS_DIR="/home1/<USER_HOME_FOLDER>/daos"
+DAOS_DIR="/home1/<PATH_TO_DAOS>/daos"
```

Comment:
option to run from SCRATCH too so remove home1
```suggestion
DAOS_DIR="/<PATH_TO_DAOS>/daos"
```